test/libsolidity/semanticTests/viaYul/various_inline_asm.sol

```diff
@@ -0,0 +1,23 @@
+contract C {
+    function f() public pure returns (uint32 x) {
+        uint32 a;
+        uint32 b;
+        uint32 c;
+        assembly {
+            function myAwesomeFunction(param) -> returnMe {
+                let localVar := 0xABC
```

Comment:
Can you use different constants such that it is easy to verify the correctness of the expectation below?